music/demos/build-demos.sh

```diff
@@ -26,7 +26,7 @@
 if [[ -n $1 ]]; then
   demos=( "${1}" )
 else
-  demos=( *.html )
+  demos=( *.ts )
```

Comment:
My thought for using .html was that we might eventually have demos with multiple .ts files, but they'll probably be launched from one html file. But I see the problem with index.html now. We could either specifically exclude it or just go with this approach until we need something more complicated. :-)